uamqp/message.py

```diff
@@ -517,9 +517,7 @@ def gather(self):
                 message_bytes = wrap_message.encode_message()
             body_size += len(message_bytes)
             if (body_size + message_size) > self.max_message_length:
-                raise ValueError(
-                    "Data set too large for a single message."
-                    "Set multi_messages to True to split data across multiple messages.")
+                raise errors.ClientMessageError("Data set too large for a single message.")
```

Comment:
It occurs to me that this is actually a breaking change. Could we do an intermediate fix for this that's non breaking - then adjust it later next time there's a major version bump?
I'm thinking, in errors.py add:
```
class MessageContentTooLarge(ValueError):
    def __init__(self):
        message = "Data set too large for a single message."
        super(ValueError, self).__init__(message)
```
Then use this error in message.py